src/main/resources/static/js/dm/viewer/ProjectViewer.js

```diff
@@ -409,6 +420,33 @@ dm.viewer.ProjectViewer.prototype.switchToProject = function(project, force) {
         this.projectController.selectProject(project);
         this.updateProjects();
         this.projectView();
+
+        $.getJSON("/store/projects/dashboard", function(users) {
+            var conflictingUsers = [];
+            for (var userId in users) {
+                if (this.currentUser.uri.indexOf(userId) < 0) {
+                    var projects = users[userId].projects;
+                    for (var i = 0; i < projects.length; i++) {
+                        if (projectUri == projects[i].uri) {
+                          conflictingUsers.push({ name: users[userId].name, email: users[userId].mail });
+                          break;
+                        }
+                    }
+                }
+            }
+            if (conflictingUsers.length > 0) {
+                returnState = this.state;
+                this.showModal(STATES.alert);
+                $("#alert-text").empty();
+                $("#alert-text").append("<p>Warning! Changes you make could be overwritten by other users. The following users are also logged in as editing this project:</p><ul id='alert-users'></ul>");
+                conflictingUsers.forEach(function(conflictingUser) {
+                  $("#alert-users").append("<li>" + conflictingUser.name + ": <a href='mailto:" + conflictingUser.email + "'>" + conflictingUser.email + "</a></li>");
+                });
+                $("#dismiss-alert").off("click").on("click", function() {
```

Comment:
Thanks @jamiefolsom -- the Continue button should leave the modal open but proceed to the TOC. Steve found the button wasn't doing anything until he cleared his browser cache, then worked correctly.